### ISSUE TITLE
On your profile, you should be able to track your comments as well as your posts

### ISSUE THREAD

**Issue by oatsandsugar** [cornelltech/unicornucopia-issue-tracker#56]

**Comment by AVatch** [comment]
keeping track of the conversations is really important and we're trying to figure out how to make this work. do you mean to just have another tab of ideas you've commented on

**Comment by ny11tamias** [comment]
I agree with that, at least we need to see the ideas we've commented on/winged from.

**Comment by oatsandsugar** [comment]
@AVatch yep, your idea would do the trick
